src/model/transaction/TransactionTypeEnum.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"),
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Copy of the TransactionType class
+ */
+
+export enum TransactionTypeEnum {
```

Comment:
please remove this enum as it is already available in `src/model/transactions/TransactionType`. 

I don't think modifying to an enum makes sense for `TransactionType` because there is a second *similar* class `TransactionVersion`. Changing to enum would make this incompatible because constants have the same name.

Reply:
Yep. noticed that, the enum is deprecated. not used anywhere, probably pulled back from merge.